src/Bridges/Constraint/bridges/fix_parametric_variables.jl

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+"""
+    FixParametricVariablesBridge{T,S} <: AbstractBridge
```

Comment:
TODOs before merging: add some more documentation here.

Add docstring to docs:
https://github.com/jump-dev/MathOptInterface.jl/blob/master/docs/src/submodules/Bridges/list_of_bridges.md#constraint-bridgesid-constraint_bridges_ref